### PR TITLE
onefetch 2.13.1

### PR DESCRIPTION
**Pull request by o2sh**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

File: Formula/onefetch.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Onefetch < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "b6fba076a2d577312cc34fb2eabaca97e4767cf822f6e7698578f624f1f4e1d8"
   end
 
+  depends_on "cmake" => :build
   depends_on "rust" => :build
 
   uses_from_macos "zlib"
```
Comment (Bo98) [inline Formula/onefetch.rb:20]: This appears to not be used anymore. Is that intentional?

**Comment by chenrui333** [comment]
why changing desc?

File: Formula/onefetch.rb (hunk 1 above)
Comment (o2sh) [reply Formula/onefetch.rb]: done.

**Comment by o2sh** [comment]
> why changing desc?

The previous description was outdated. the updated one is the one use for  [`crates`](https://github.com/o2sh/onefetch/blob/d5fbb3dc797aec968f8ae5e33cbfb4ae30cadb9e/Cargo.toml#L4), [`snap`](https://github.com/o2sh/onefetch/blob/d5fbb3dc797aec968f8ae5e33cbfb4ae30cadb9e/snap/snapcraft.yaml#L4), and debian.

I just wanted to be consistent, but I can revert if it blocks the PR.

File: Formula/onefetch.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Onefetch < Formula
-  desc "Git repository summary on your terminal"
+  desc "A command-line Git information tool"
```
Comment (SMillerDev) [inline Formula/onefetch.rb:2]:
```suggestion
  desc "Command-line Git information tool"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> > why changing desc?
> 
> The previous description was outdated. the updated one is the one used for [`crates`](https://github.com/o2sh/onefetch/blob/d5fbb3dc797aec968f8ae5e33cbfb4ae30cadb9e/Cargo.toml#L4), [`snap`](https://github.com/o2sh/onefetch/blob/d5fbb3dc797aec968f8ae5e33cbfb4ae30cadb9e/snap/snapcraft.yaml#L4) and Github (about section).
> 
> I just wanted to be consistent, but I can revert if it blocks the PR.

sounds good 🚀

**Pull request closed without merging**